<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>708</width>
    <height>356</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="restartButton">
    <property name="geometry">
     <rect>
      <x>502</x>
      <y>272</y>
      <width>174</width>
      <height>52</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(214, 0, 0);
border-radius: 6px;
color: #FFFFFF;
font: 500 16px &quot;Roboto&quot;;</string>
    </property>
    <property name="text">
     <string>Перезагрузить
сейчас</string>
    </property>
   </widget>
   <widget class="QPushButton" name="waitButton">
    <property name="geometry">
     <rect>
      <x>345</x>
      <y>272</y>
      <width>133</width>
      <height>52</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
border: 2px solid #D60000;
border-radius: 6px;
color: #D60000;
font: 500 16px &quot;Roboto&quot;;</string>
    </property>
    <property name="text">
     <string>Отложить
на 3 дня</string>
    </property>
   </widget>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>256</width>
      <height>356</height>
     </rect>
    </property>
    <property name="whatsThis">
     <string>position: absolute;
width: 256px;
height: 356px;
left: 0px;
top: 0px;

background: linear-gradient(180deg, #FE4545 0%, #E10404 100%);</string>
    </property>
    <widget class="QGraphicsView" name="graphicsView">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>256</width>
       <height>356</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">gridline-color: qlineargradient(spread:pad, x1:0.25, y1:0.267045, x2:1, y2:1, stop:0 rgba(254, 69, 69, 255), stop:1 rgba(225, 4, 4, 255));
</string>
     </property>
     <property name="backgroundBrush">
      <brush brushstyle="NoBrush">
       <color alpha="255">
        <red>255</red>
        <green>102</green>
        <blue>0</blue>
       </color>
      </brush>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>75</x>
       <y>20</y>
       <width>105</width>
       <height>105</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;img src=&quot;:/icons/icons/Frame.png&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>16</x>
       <y>132</y>
       <width>224</width>
       <height>224</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;img src=&quot;:/icons/icons/pc.png&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
     <property name="indent">
      <number>-1</number>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_static">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>32</y>
      <width>420</width>
      <height>28</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 700 24px &quot;Roboto&quot;;
color: rgb(33, 33, 33);</string>
    </property>
    <property name="text">
     <string>Ваш компьютер не перезагружался</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_days">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>64</y>
      <width>242</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 700 34px &quot;Roboto&quot;;
color: rgb(33, 33, 33);</string>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_description">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>124</y>
      <width>396</width>
      <height>48</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 16px &quot;Roboto&quot;;
color: rgb(101, 101, 101);</string>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_static_2">
    <property name="geometry">
     <rect>
      <x>402</x>
      <y>0</y>
      <width>160</width>
      <height>28</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 700 16px &quot;Roboto&quot;;
color: rgb(214, 0, 0);</string>
    </property>
    <property name="text">
     <string>ВсеИнструменты.ру</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>256</x>
      <y>25</y>
      <width>452</width>
      <height>5</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(33, 33, 33);</string>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
